CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# v0.2.2 dbt_qualtrics_source
```

Comment:
```suggestion
# dbt_qualtrics_source v0.2.2
```
Updating for compatibility with the auto releaser.

Reply:
Updated